common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -308,13 +308,17 @@ protected void renewAccessToken(@NonNull final SilentTokenCommandParameters para
                 final String subErrorCode = tokenResult.getErrorResponse().getSubError();
                 Logger.info(TAG, "Error: " + errorCode + " Suberror: " + subErrorCode);
 
-                if (errorCode.equals(INVALID_GRANT) && subErrorCode.equals(BAD_TOKEN)) {
+                if (null != errorCode && errorCode.equals(INVALID_GRANT)
+                        && null != subErrorCode && subErrorCode.equals(BAD_TOKEN)) {
```

Comment:
INVALID_GRANT.equals(errorCode) && BAD_TOKEN.equals(subErrorCode) ?

Reply:
Yeah let's flip it